project_material/models/stock_warehouse.py

```diff
@@ -193,12 +204,8 @@ def _get_consumption_route_values(self):
             'warehouse_ids': [(4, self.id)],
         }
 
-    def _create_consumption_route(self):
-        vals = self._get_consumption_route_values()
-        vals['pull_ids'] = [
-            (0, 0, self._get_consumption_pull_values()),
-        ]
-        self.consu_route_id = self.env['stock.location.route'].create(vals)
+    def _get_consumption_route_description(self):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/manual/numigi/odoo-project-addons/pullRequest?prid=4797804)